handbook/quality/README.md

```diff
@@ -0,0 +1,146 @@
+# Quality
+
+## Human-oriented QA
+
+Fleet uses a human-oriented quality assurance (QA) process to make sure the product meets the standards of users and organizations.
+
+Automated tests are important, but they can't catch everything. Many issues are hard to notice until a human looks empathetically at the user experience, whether in the user interface, the REST API, or the command line.
+
+The goal of quality assurance is to catch unexpected behavior before release:
+- Bugs
+- Edge cases
+- Error message UX
+- Developer experience using the API/CLI
+- Operator experience looking at logs
+- API response time latency
+- UI comprehensibility
+- Simplicity
+- Data accuracy
+- Perceived data freshness
+- Product’s ability to save users from themselves
+
+## Collecting bugs
+
+To try Fleet locally for QA purposes, run `fleetctl preview`, which defaults to running the latest stable release.
+
+To target a different version of Fleet, use the `--tag` flag to target any tag in [Docker Hub](https://hub.docker.com/r/fleetdm/fleet/tags?page=1&ordering=last_updated), including any git commit hash or branch name. For example, to QA the latest code on the `main` branch of fleetdm/fleet, you can run: `fleetctl preview --tag=main`
+
+To start preview without starting the simulated hosts, use the `--no-hosts` flag (e.g., `fleetctl preview --no-hosts`).
+
+All QA steps should be possible using `fleetctl preview`. Please refer to [docs/Contributing/Testing-and-local-development.md](https://fleetdm.com/docs/contributing/testing) for flows that cannot be completed using `fleetctl preview`.
```

Comment:
```suggestion
```

I think we can remove this section. 

It's outdated because I removed documentation for the "QA steps." It looks like I forgot to remove this section when I removed the QA steps: https://github.com/fleetdm/fleet/pull/7669/files#diff-5f7714f6b120efb9253d8ce52b7fe87629fe2b4f5d515a900c4f68dfe6336d34